Uplift/Views/HomeView.swift

```diff
@@ -71,18 +71,33 @@ struct HomeView: View {
             if let gyms = viewModel.gyms {
                 VStack(spacing: 12) {
                     HStack(spacing: 12) {
-                        capacityCircle(facility: gyms.facilityWithName(name: Constants.FacilityNames.hnhFitness))
+                        capacityCircleNavLink(
+                            fitnessCenterName: Constants.FacilityNames.hnhFitness,
+                            gyms: gyms
```

Comment:
You should not need to pass in `gyms` since the function you defined below already has access to `viewModel.gyms`.